medusa/providers/torrent/html/tvroad.py

```diff
@@ -0,0 +1,189 @@
+
+            # Continue only if at least one release is found
+            if len(torrent_rows) < 2:
+                log.debug('Data returned from provider does not contain any torrents')
+                return items
+
+            # Catégorie, Release, Date, DL, Size, C, S, L
+            labels = [label.get_text(strip=True) for label in torrent_rows[0]('th')]
+
+            # Skip column headers
+            for row in torrent_rows[1:]:
+                cells = row('td')
+                if len(cells) < len(labels):
+                    continue
+
+                try:
+                    title = cells[labels.index('Nom')].get_text(strip=True)
+                    download = cells[labels.index('DL')].find('a')['href']
+                    download_url = download
+                    print(download_url)
+                    if not all([title, download_url]):
+                        continue
+
+                    seeders = try_int(cells[labels.index('S')].get_text(strip=True))
+                    leechers = try_int(cells[labels.index('L')].get_text(strip=True))
+
+                    # Filter unseeded torrent
+                    if seeders < self.minseed:
+                        if mode != 'RSS':
+                            log.debug("Discarding torrent because it doesn't meet the"
+                                      ' minimum seeders: {0}. Seeders: {1}',
+                                      title, seeders)
+                        continue
+
+                    size_index = labels.index('Size') if 'Size' in labels else labels.index('Taille')
+                    torrent_size = cells[size_index].get_text()
+                    size = convert_size(torrent_size, units=units) or -1
+
+                    item = {
+                        'title': title,
+                        'link': download_url,
+                        'size': size,
+                        'seeders': seeders,
+                        'leechers': leechers,
+                        'pubdate': None,
+                    }
+                    if mode != 'RSS':
+                        log.debug('Found result: {0} with {1} seeders and {2} leechers',
+                                  title, seeders, leechers)
+
+                    items.append(item)
+                except (AttributeError, TypeError, KeyError, ValueError, IndexError):
+                    log.exception('Failed parsing provider.')
+
+        return items
+
+    def login(self):
+        """Login method used for logging in before doing search and torrent downloads."""
+        if any(dict_from_cookiejar(self.session.cookies).values()):
+            return True
+
+        login_params = {
+            'username': self.username,
+            'password': self.password,
+        }
+
+        response = self.session.post(self.urls['login'], data=login_params)
+
+        if not response: # can't check response.text as POST request only return text when fail
```

Comment:
You can use `if not response or not response.text` because in case response is missing, it will stop at the first condition.